src/helpers/getSectionColors.ts

```diff
@@ -0,0 +1,28 @@
+/* Retrieves section color values.
+ *
+ * Requires a section type as defined by sectionDataMap. Accepts an optional color
+ * preference ("primary" or "secondary").
+ *
+ * @returns An object containing both primary and secondary colors (default) or a string of the value requested
+ * in the optional color preference.
+ */
+import { sectionDataMap, SectionTypes } from "./types";
+
+type ColorSection = {
+  primary: string;
+  secondary: string | null;
+};
+
+type ColorVal = string;
+
+export function getSectionColors(
+  type: SectionTypes,
+  colorVal?: "primary" | "secondary"
+): ColorSection | ColorVal {
+  let colorValOrSection = sectionDataMap[type];
+
+  if (colorVal) {
+    colorValOrSection = colorValOrSection[colorVal];
+  }
+  return colorValOrSection;
+}
```

Comment:
This function needs to account for both light mode and dark mode values.  Let's setup a call to talk about the best way to handle this.

Cc @EdwinGuzman

Reply:
Can we make this a new ticket to work on for a later iteration? We don't need the dark mode variant right now and I don't want this to be a blocker for its initial release.

Reply:
For now I simply changed the default corder color to use the blogs color since that uses the `ui.default.border` color.
Once it is decided, it can be updated accordingly.

Reply:
Ok, thanks.  That works.  I see no reason to change that in the future.  Please add a comment to the code that I explains why the default is `"blog"`.

As for the dark mode support in the `getSectionColors` function, yes, let's create a new ticket to add that functionality in the next iteration of the component.

Reply:
New ticket to update the function: https://jira.nypl.org/browse/DSD-1604